### PR TITLE
hebrew test

### PR DESCRIPTION
**Pull request by vvvmax**
Maybe do you have idea how to see hebrew test right? 
This is a big issue in windows. haven't found normal editor that supports rtl. 
anyway, I can overcome it and edit the files in rtl text area, but in githubs it don't show in rtl eather. 
Do you know how to see the file in rtl in github as well?

### CODE REVIEW

File: maxtest.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+קו מפריד
+```javascript
+alert("Hello World")
+```
+```php
+echo("Hello World")
+```
+
+- [לינק ראשון](http://www.fsf.org/)
```
Comment (ynonp) [inline maxtest.md]: <div dir="rtl">
אפשר לכתוב עברית מימין לשמאל אם שמים אותה בתוך דיב מתאים
</div>

Comment (vvvmax) [inline maxtest.md]: אוקיי אם לא מפריע לך אז בסדר. עכשיו רק נשאר לי להבין איך אני מעלה קבצים
מהדיסק ולא מהממשק של הגיטהאב ואפשר להתחיל עם התכנים. דרך אגב, לא מצאתי עורך
טקסט מתאים עם תמיכה בrtl לווינדווס :( במאק דווקה יש.

Comment (ynonp) [inline maxtest.md]: Hi,

I think it'll be best to have the text itself in Hebrew, but the discussions on it in English (like this one). This way Github won't get confused by directions.

Re. an editor you can use the online RTL markdown editor here (and copy-paste the result):
http://dariubs.github.io/rtlmd/

Or learn how to use Emacs:
https://www.gnu.org/software/emacs/

(it takes a few days to get to a working condition but I think it's worth it. Talk to me if you need help)

**Comment by vvvmax** [comment]
Oh, I begin to understand about how github works :) it's all about making a branch, edit it collaborated, and then push it to the master. 
I just thought that the branch should appear in another folder aside the master branch, but it simse that it's always stays in it's place. Now it's more clear to me. 

So I'll add new files here, please check and merge to master (or talk to me about the changes) when you have time.

**Pull request closed without merging**